server/src/index.ts

```diff
@@ -12,6 +16,16 @@ const emailer = getEmailer(
   config.getEmailerFromAddress()
 );
 
+
+mongoose.connect(config.getDbUri(),(err) => {
+  if (err){
+    console.log(err)
+  }
+  else {
+    console.log('Connected to MongoDB')
+  }
+})
```

Comment:
I think this should be moved into a function in the database module that takes an IConfig argument